apps/omg_watcher/test/omg_watcher/api/account_test.exs

```diff
@@ -13,24 +13,42 @@
 # limitations under the License.
 
 defmodule OMG.Watcher.API.AccountTest do
-  use ExUnitFixtures
   use ExUnit.Case, async: false
-  use OMG.DB.Fixtures
 
   alias OMG.TestHelper
+  alias OMG.Utxo
   alias OMG.Watcher.API.Account
 
+  require Utxo
+
   @eth OMG.Eth.zero_address()
   @payment_output_type OMG.WireFormatTypes.output_type_for(:output_payment_v1)
 
+  setup do
```

Comment:
+1 on moving fixture to setup, this makes it more explicit, plus avoids deep fixture inheritance that fixtures allow